Resources/Private/JavaScript/asset-collections/src/fragments/assetCollection.ts

```diff
@@ -1,13 +1,22 @@
 import { gql } from '@apollo/client';
-import { TAG_FRAGMENT } from './tag';
+import { TAG_FRAGMENT } from '@media-ui/feature-asset-tags/src/fragments/tag';
 
 export const ASSET_COLLECTION_FRAGMENT = gql`
     fragment AssetCollectionProps on AssetCollection {
         id
         title
+        parent {
+            id
+            title
+        }
+        children {
+            id
+            title
+        }
```

Comment:
Always fetching parent and all children makes this query really really expensive. One of the two should be enough per usecase.
Similar thing for `tags` and `assetCount` do we really need to fetch those for every asset all the time?